Chummer/Classes/clsXmlManager.cs

```diff
@@ -1857,6 +1857,69 @@ public bool Equals(CustomDataDirectoryInfo other)
             return other != null && Name == other.Name && Path == other.Path;
         }
 
+        /// <summary>
+        /// Reads in an Manifest.xml in the given directory and returns it's language specific description. Defaults to an description without any attributes.
+        /// </summary>
+        /// <param name="strDirectory"></param>
+        /// <returns>A description as String, adds a comment if it defaulted to a node without language attribute</returns>
+        public string GetManifestData(string strDirectory)
+        {
+            string strLanguage = GlobalOptions.Language;
+            XmlDocument xmlObjManifest = new XmlDocument();
```

Comment:
Have you profiled this? I imagine you'd ideally want to only create and load in the `XmlDocument` once, either at the first request for the manifest or when the Info itself is created/loaded. Constantly re-creating the manifest text every single time a description is requested is not ideal.

Reply:
It's called once, for every custom dataset, when opening the Character Options.
I could implement a `Description` property for the `ManifestInfo` class and store the tooltip there.
Then a simple `if`, would check if the property was already set.

Reply:
I just remembered why I didn't go with a property in the first place.
Calling `GetManifestData()` in the constructor, leads to an `standard text style has to be set before win32 object can be created` exception. After that I just scraped the idea all together.

That said I will just not do it in the constructor of `CustomDataDirectoryInfo`, but in the load of the `frmCharacterOptions` and just check if the property is set already.